### ISSUE TITLE
Support language tool language auto detection

### ISSUE THREAD

**Issue by oblitum** [sylvainhalle/textidote#151]
LanguageTool language auto detection:

```
  -adl, --autoDetect       auto-detect the language of the input text - note this will not detect
                           variants like 'English (US)', so you will not get spell checking for
```

It would be great if TeXtidote provided a means to use that instead of only supporting hardcoded language.

**Comment by 73** [comment]
That would be great indeed. I often find myself missing to provide `--check`.

**Comment by sylvainhalle** [comment]
Well, not providing `--check` is a different matter. TeXtidote can still verify things in a text without checking spelling and grammar, so the parameter is likely to remain mandatory.

**Comment by 73** [comment]
But the language identifier could go. I write texts in two languages and can not reuse my config easily. So it would be just `--check` instead of `--check en|fr|es|...`

**Comment by sylvainhalle** [comment]
Yes, that's what I plan to implement. The option is probably going to be called `--autocheck` without any parameter.

**Comment by 73** [comment]
May I propose something more specific? `--spelling` or `--spellcheck`?
Because all textidote does is checking.
